packages/flutter/test/material/chip_test.dart

```diff
@@ -6167,6 +6167,96 @@ void main() {
       SystemMouseCursors.forbidden,
     );
   });
+
```

Comment:
To ensure the custom hit test and the semantic bounds match up and we haven't regressed anything, can you add a test where the delete button is larger than 48x48 - if that is possible? I am less familiar with this widget, and it is surprisingly complex. :)

Reply:
I will try, but I'm not confident for the moment that we can get the hit test and semantics bounds always matching (the calculation for the hit logic area is complex. Looking at the history, It looks like Greg tried hard to get a logic that fits most of the use cases and that it was not an easy task). Let me experiment and share my findings.

> 'Delete button semantic tap target can be larger than the minimum interactive dimension' - if that is possible?

Yes, it is possible, for instance by increasing the label font size and the icon size (I did it in one of the test I added in the second commit for this PR). I will also try to see the effect of increasing text scaling.